### DISCUSSION TITLE
Artifacts in chrome based application

### DISCUSSION THREAD

**Discussion by miguelgsousa** [ChefKissInc/NootedRed#202] [Q&A] [answered]
There has been some progress in resolving artifacts caused by programs like Chrome, Chromium-based browsers and apps like Sublime Text? It's the only thing that stops me from using my Hackintosh frequently because I depend on a specific app that in its latest version causes artifacts

**Comment by VisualEhrmanntraut** [comment, marked as answer]
Not yet. See #158
